lib/Client.js

```diff
@@ -918,9 +918,12 @@ class Client extends EventEmitter {
     * Edit a guild channel's position. Note that channel position numbers are lowest on top and highest at the bottom.
     * @arg {String} channelID The ID of the channel
     * @arg {Number} position The new position of the channel
+    * @arg {Object} options Additional options when editing position
+    * @arg {Boolean} options.lockPermissions Whether to sync the permissions with the new parent if moving to a new category
+    * @arg {String} options.parentID The new parent ID (category channel) for the channel that is moved
     * @returns {Promise}
     */
-    editChannelPosition(channelID, position) {
+    editChannelPosition(channelID, position, options) {
```

Comment:
Could probably deprecate the `position` param and move it into `options` like how we do for `createChannel`

Reply:
Default param also needed otherwise this'll throw a typeerror

Reply:
position being mandatory anyway, I think we are good keeping there. Avoid an unecessary breaking change as well.
Missed that default param though ty